wallet-new/integration/Types.hs

```diff
@@ -0,0 +1,88 @@
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE ConstraintKinds #-}
+
+module Types
+    ( Probability
+    , createProbability
+    , getProbability
+    , Action (..)
+    , WalletState (..)
+    , TestResult (..)
+    , ActionWalletState
+    , ActionProbabilities
+    , WIntTestMode
+    ) where
+
+import Universum
+
+import Cardano.Wallet.API.V1.Types (Wallet)
+
+type WIntTestMode m =
```

Comment:
I'm not very sold to this "url-shortened" names: I would personally prefer `IntegrationTestMode` or simply `WalletTestMode`, (or even just `TestMode`), but `WIntTestMode` doesn't communicate a great deal more and that's just noise at this stage (my 2 cents anyway).

Reply:
I agree, but the name I came up with was a bit too long. The problem is how to pack information in few words. Will fix.